tests/gym_test.py

```diff
@@ -0,0 +1,36 @@
+import unittest
+import responses
+from pittapi import gym
+
+GYM_URL = "https://connect2concepts.com/connect2/?type=bar&key=17c2cbcb-ec92-4178-a5f5-c4860330aea0"
```

Comment:
No need to declare this again, you can simply access it from the gym module :)

gym.GYM_URL